FirebaseVertexAI/Tests/TestApp/Tests/Integration/GenerateContentIntegrationTests.swift

```diff
@@ -0,0 +1,118 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import FirebaseAuth
+import FirebaseCore
+import FirebaseStorage
+import FirebaseVertexAI
+import Testing
+import VertexAITestApp
+
+@testable import struct FirebaseVertexAI.APIConfig
+
+@Suite(.serialized)
+struct GenerateContentIntegrationTests {
+  static let vertexV1Config = APIConfig(service: .vertexAI, version: .v1)
+  static let vertexV1BetaConfig = APIConfig(service: .vertexAI, version: .v1beta)
+  static let developerV1BetaConfig = APIConfig(
+    service: .developer(endpoint: .generativeLanguage),
+    version: .v1beta
+  )
```

Comment:
What happens with `.v1` for developer API? (Sorry if you've already explained)

Reply:
It's actually functional right now since it's temporarily going directly to the `generativelanguage.googleapis.com` endpoint but as soon as we go through the proxy it'll only support `v1beta` at first.

Reply:
Correction: `tools`, `toolConfig`, `responseMIMEType` and `systemInstruction` are all unavailable on v1 in the Developer API. Will re-evaluate whether to keep these in the default config in the integration tests in a follow-up.